modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -589,37 +589,86 @@ public Map<String, Object> getSystemModelAttributes(
 			HashMapBuilder.<String, Object>put(
 				"createDate",
 				GetterUtil.get(
-					baseModelAttributes.get("createDate"), objectEntryId)
+					baseModelAttributes.get("createDate"), primaryKey)
 			).put(
 				"externalReferenceCode",
 				GetterUtil.get(
 					baseModelAttributes.get("externalReferenceCode"),
-					objectEntryId)
+					primaryKey)
 			).put(
 				"modifiedDate",
 				GetterUtil.get(
-					baseModelAttributes.get("modifiedDate"), objectEntryId)
+					baseModelAttributes.get("modifiedDate"), primaryKey)
 			).put(
 				"objectDefinitionId", objectDefinition.getObjectDefinitionId()
 			).put(
 				"uuid",
-				GetterUtil.get(baseModelAttributes.get("uuid"), objectEntryId)
+				GetterUtil.get(baseModelAttributes.get("uuid"), primaryKey)
 			).build();
 
 		for (ObjectField objectField :
 				_objectFieldLocalService.getObjectFields(
 					objectDefinition.getObjectDefinitionId())) {
 
+			if (!objectField.isSystem()) {
```

Comment:
I'm actually removing the non-system fields because I'm adding them right after (in this [line](https://github.com/liferay-forms/liferay-portal/blob/a176df4fa0d69ff68a2107a8e3e7c85271793fe8/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java#L624)). 

Extended properties are custom field values for system objects.

Reply:
Oh it makes sense. Thanks, @carolmariaabb